modules/web/src/app/node-data/basic/provider/nutanix/template.html

```diff
@@ -102,6 +106,7 @@
                      [formControlName]="Controls.CategoryValue"
                      [label]="categoryValuesLabel[getSelectedCategory(control)] || CategoryValueState.Empty"
                      [isDisabled]="control.get(Controls.Category).invalid"
+                     [disablekmValueChangedIndicator]='true'
```

Comment:
I think we don't have `disablekmValueChangedIndicator` Input in km-combobox component.
```suggestion
                     [enableKmValueChangedIndicator]="false"
```